packages/roosterjs-editor-plugins/lib/plugins/Announce/AnnouncePlugin.ts

```diff
@@ -65,23 +84,52 @@ export default class Announce implements EditorPlugin {
     dispose() {
         this.ariaLiveElement?.parentElement?.removeChild(this.ariaLiveElement);
         this.ariaLiveElement = undefined;
+        this.stringsMapOrGetter = undefined;
+        this.lastFocusedElement = null;
+        while (this.features.length > 0) {
+            this.features.pop();
+        }
+        this.editor = undefined;
     }
 
     /**
      * Handle events triggered from editor
      * @param event PluginEvent object
      */
-    onPluginEvent(event: PluginEvent) {
+    onPluginEvent(ev: PluginEvent) {
         if (
             this.editor &&
-            event.eventType == PluginEventType.ContentChanged &&
-            event.additionalData?.getAnnounceData
+            ev.eventType == PluginEventType.ContentChanged &&
+            ev.additionalData?.getAnnounceData
         ) {
-            const data = event.additionalData.getAnnounceData();
+            const data = ev.additionalData.getAnnounceData();
             if (data) {
                 this.announce(data, this.editor);
             }
         }
+
+        if (ev.eventType == PluginEventType.KeyDown && this.editor) {
+            this.handleFeatures(ev, this.editor);
+        }
+    }
+
+    private handleFeatures(event: PluginKeyDownEvent, editorInput: IEditor) {
+        editorInput.runAsync(editor => {
+            if (!editor || editor.isDisposed()) {
```

Comment:
minor: when use this editor parameter, it is guaranteed that editor must not be disposed. So no need to check.

Reply:
I see, removed this check since it is not needed. Thanks.